hedera-node/hedera-token-service-impl/src/main/java/com/hedera/node/app/service/token/impl/WritableAccountStore.java

```diff
@@ -170,4 +171,39 @@ public Set<AccountID> modifiedAccountsInState() {
     public Set<Bytes> modifiedAliasesInState() {
         return aliases().modifiedKeys();
     }
+
+    /**
+     * Gets the original value associated with the given accountId before any modifications were made to
+     * it. The returned value will be {@code null} if the accountId does not exist.
+     *
+     * @param id The accountId. Cannot be null, otherwise an exception is thrown.
+     * @return The original value, or null if there is no such accountId in the state
+     * @throws NullPointerException if the accountId is null.
+     */
+    @Nullable
+    public Account getOriginalValue(@NonNull final AccountID id) {
```

Comment:
Will it _ever_ be appropriate for us to have used a `0.0.<alias>` account id to modify a `WritableKVState`?